SWI-cpp.h

```diff
@@ -130,6 +130,9 @@ class PlTerm
   term_t ref;
 
   PlTerm();
+  PlTerm(const PlTerm &other)
+  { ref = other.ref;
+  }
```

Comment:
If you are only targeting C++ 11 and greater, the following is more succinct:
```suggestion
  PlTerm(const PlTerm &other) = default;
```

Reply:
This is still applicable even with my recent update to the PR.